### DISCUSSION TITLE
Love the spinning globe what plugin did you use?

### DISCUSSION THREAD

**Discussion by redimongo** [dubinc/dub#216] [Q&A]
Just trying to find out what tool you used for the globe on the homepage. Love how this is done, and think it could work for one of my projects.

**Comment by fmerian** [comment]
hi @redimongo 👋 

@steven-tey please correct me if I'm wrong -- I think the globe was built with @shuding's OSS [cobe.vercel.app](https://cobe.vercel.app)

hope it helps!

**Reply by steven-tey** [reply]
Yes, that's correct! Thanks for answering @fmerian!
